app/processing/siti-agri/handle-known-defects/correct-invalid-due-dates/invalid-due-dates.js

```diff
@@ -25,4 +25,49 @@ module.exports = [{
 }, {
   invalid: '2023-03-05',
   correction: '2023-06-05'
+}, {
```

Comment:
Is there a reason that we set our `invalid` dates to be on the 5th of each month `2022-08-05` but the table in the user story states first of the month.
<img width="299" alt="image" src="https://user-images.githubusercontent.com/16856615/223658121-47a22826-e8af-4824-a88a-dd6fbb549a59.png">

Reply:
The agreement start dates are the 1st, Siti Agri maps them to the 5th.   But the 5th of the wrong month.  You're right though it's not quite clear in the table what the ask is.  But 5th -> 5th is correct as per existing mapping